core-tests/src/test/java/org/mule/runtime/core/processor/strategy/AbstractProcessingStrategyTestCase.java

```diff
@@ -288,27 +314,26 @@ public void asyncCpuLightConcurrent() throws Exception {
   }
 
   protected void internalAsyncCpuLightConcurrent(boolean blocks) throws MuleException, InterruptedException {
```

Comment:
This is very similar to `internalConcurrent`. Can the common parts be extracted?